package-lock.json

```diff
@@ -12,8 +12,11 @@
         "autoprefixer": "^10.2.5",
         "eslint": "^7.26.0",
         "prettier": "^2.3.2",
-        "sass": "^1.26.5",
+        "sass": "^1.69.7",
         "vite": "^2.2.3"
+      },
+      "peerDependencies": {
+        "sass": ">=1.33.0"
```

Comment:
I’m not sure this should be there, as technically the package also has a CSS-only output that would work fine without Sass or with an older Sass version.

Reply:
My reading of `peerDependencies` was that it was just "if `sass` is installed, make sure it's at least 1.33", as opposed to explicitly installing it. I intentionally went with the earliest version which supports `sass:math` so it's likely most people will already be on the version, and given it's over 3 years old, the chance of people already being on a supported version is quite high.

Reply:
No, at least npm will automatically install a package’s `peerDependencies` on install. There is a way to mark peer dependencies as optional like what you describe with `peerDependenciesMeta`, but I don’t use this often enough to know whether that’s more appropriate. And it’d still not be correct to say the package only supports `sass` above a certain version because of what I said about the CSS version.

Reply:
Let's remove it from `peerDependencies` and add it to [the list of requirements in the README](https://github.com/torchbox/tbxforms#requirements), as we do with `wagtail`.

We only support `wagtail>=2.15` though [wagtail is not listed as a dependency](https://github.com/torchbox/tbxforms/blob/main/pyproject.toml#L55). Installing `tbxforms` in a project using a `wagtail<2.15` causes an error, and there haven't been any associated issues or confusion so far as the requirements are explicit.